kube/elasticsearch/elasticsearch-config.yaml

```
@@ -0,0 +1,16 @@
+  name: elasticsearch-config
+data:
+  elasticsearch.yml: |
+    xpack.security.enabled: true
+    xpack.license.self_generated.type: basic
+    network.host: 0.0.0.0
+    path.repo: ["/data/es_snapshot_repository"]
+    s3.client.default.endpoint: "s3.ap-northeast-1.amazonaws.com"
+    s3.client.default.protocol: https
+    s3.client.default.read_timeout: 50s
+    s3.client.default.max_retries: 3
+    s3.client.default.use_throttle_retries: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)